packages/web-react/src/components/Accordion/demo/AccordionStayOpen.tsx

```diff
@@ -12,7 +12,7 @@ import { Link } from '../../Link';
 import { Pill } from '../../Pill';
 import { content } from './Accordion';
 
-const Story: ComponentStory<typeof Accordion> = () => {
+const Story: ComponentStory<typeof Accordion> | ElementType = () => {
```

Comment:
In the file abore, you are removing `ComponentStory` type. Here, you are adding `ElementType`. Why? Both should be the same, shouldn't be? Or where is the problem that I do not see... 🤔

Reply:
i was fighting with types, forgot to remove this one... removed